clients/apps/web/src/components/Settings/OrganizationSubscriptionSettings.tsx

```diff
@@ -127,6 +128,28 @@ const OrganizationSubscriptionSettings: React.FC<
               )}
             />
           </SettingsGroupItem>
+
+          <SettingsGroupItem
```

Comment:
I don't think we should let users choose this. Can we consider that existing merchants keep the current behaviour and new merchants get the customer-sequence?